package.json

```diff
@@ -27,9 +27,12 @@
     "@folio/stripes": "^9.0.0",
     "@folio/stripes-cli": "^3.0.0",
     "@formatjs/cli": "^6.1.3",
+    "babel-jest": "^26.3.0",
     "core-js": "^3.6.1",
     "eslint": "^7.32.0",
     "eslint-plugin-filenames": "^1.3.2",
+    "eslint-plugin-import": "^2.22.1",
```

Comment:
I think this package is available from "@folio/eslint-config-stripes". Do you need to explicitly add it?

Reply:
Updated